app/code/Magento/InventoryCatalog/Plugin/CatalogInventory/Model/ResourceModel/Stock/Status/AdaptAddIsInStockFilterToCollectionPlugin.php

```diff
@@ -53,7 +58,12 @@ public function aroundAddIsInStockFilterToCollection(
         $collection
     ) {
         $stockId = $this->getStockIdForCurrentWebsite->execute();
-        $this->addIsInStockFilterToCollection->execute($collection, $stockId);
+        if ($this->defaultStockProvider->getId() === $stockId) {
+            $proceed($collection);
```

Comment:
Should it be returned?

Reply:
Overridden function is not returning any thing for above class but I agree to returned in `AdaptAddStockStatusToSelectPlugin` and `AdaptAddIsInStockFilterToCollectionPlugin` classes